src/components/ErrorPage.tsx

```diff
@@ -1,59 +1,75 @@
-import { Box, Button, Center, Flex, Heading, Link } from "@chakra-ui/react";
+import { Box, Button, Center, Link, VStack } from "@chakra-ui/react";
+
+import { useOffboardingModal } from "./Offboarding/useOffboardingModal";
+import { ModalContentWrapper } from "./Onboarding/ModalContentWrapper";
+import { downloadBackupFile } from "./Onboarding/restoreBackupFile/utils";
+import { NoticeIcon, ReloadIcon } from "../assets/icons";
+import BackgroundImage from "../assets/onboarding/background_image.png";
+import colors from "../style/colors";
 
 const feedbackEmailBodyTemplate =
   "What is it about? (if a bug report please consider including your account address) %0A PLEASE FILL %0A%0A What is the feedback? %0A PLEASE FILL";
 
-// TODO: prepare for Beta release properly
 export const ErrorPage: React.FC = () => {
+  const { modalElement: OffboardingModal, onOpen: onOpenOffboardingModal } = useOffboardingModal();
+
   const onRefresh = () => {
     window.location.reload();
   };
 
-  const onBackup = () => {
-    const jsonString = `data:text/json;chatset=utf-8,${encodeURIComponent(
-      JSON.stringify(window.localStorage)
-    )}`;
+  return (
+    <Center height="100vh" padding="60px" backgroundImage={BackgroundImage} backgroundSize="cover">
+      <Box
+        width="100%"
+        maxWidth="460px"
+        height="100%"
+        maxHeight="480px"
```

Comment:
I don't think we should have a maxHeight at all because what if we exceed it? it should grow naturally with the amount of content it holds (unless we want to have a scrollbar)